fertiscan/db/queries/errors.py

```diff
@@ -0,0 +1,606 @@
+"""
+Query exceptions hierarchy:
+
+    Exception
+    |__QueryError
+       |__InspectionQueryError
```

Comment:
Could we just add another tier for each object? I think it would helps classifying the errors? WDYT?

Reply:
How do you mean?

Reply:
```Python
"""
Exception
    |__QueryError
       |__Inspection
          |__InspectionQueryError
          |__InspectionXXXXXError
       |__Label_Information
          |__Label_Specific_errors
       |__Metrics
          |__...
"""
```

Reply:
I feel like having all the Query errors under the umbrella of querry error is not specific enough. 
Classifying the errors in smaller groups within the Query Error type seems to be more instinctive to me (having errors per objects)

Reply:
Please note that the actual hierarchy is:
```
    Exception
    |__QueryError
       |__InspectionQueryError
          |__InspectionNotFoundError
          |__InspectionCreationError
          |__...
...
```

Hence the clarification:
```
Each QueryError sub type exception has specific errors related to creation, retrieval, updating, and deletion, as well as a 'not found' error.
```

Reply:
Thank you for the clarification. From my understanding of the file Comment it was not the case.

Reply:
I updated the note.